packages/extension-ui/src/Popup/Derive/SelectParent.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019-2020 @polkadot/extension-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React, { useCallback, useContext, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+import { AccountContext, ActionContext, ButtonArea, Checkbox, InputWithLabel, NextStepButton, VerticalSpace } from '../../components';
+import AddressDropdown from './AddressDropdown';
+import { validateAccount } from '../../messaging';
+import { DerivationPath } from '../../partials';
+import { nextDerivationPath } from '../../utils/nextDerivationPath';
+
+interface Props {
+  parentAddress: string;
+  onDerivationConfirmed: (derivation: { account: { address: string; suri: string }; parentPassword: string }) => void;
+}
+
+export function SelectParent ({ onDerivationConfirmed, parentAddress }: Props): React.ReactElement<Props> {
+  const onAction = useContext(ActionContext);
+  const { accounts, hierarchy } = useContext(AccountContext);
+  const [account, setAccount] = useState<null | { address: string; suri: string }>(null);
+  const [parentPassword, setParentPassword] = useState<string>('');
+  const [isProperParentPassword, setIsProperParentPassword] = useState(false);
+  const [shouldAccountBeDerived, setIfAccountShouldBeDerived] = useState(true);
```

Comment:
```suggestion
  const [shouldAccountBeDerived, setShouldAccountBeDerived] = useState(true);
```
For consistency with `useState` pattern.